fdbclient/Knobs.cpp

```diff
@@ -51,6 +51,7 @@ ClientKnobs::ClientKnobs(bool randomize) {
 	init( DEFAULT_MAX_BACKOFF,                     1.0 );
 	init( BACKOFF_GROWTH_RATE,                     2.0 );
 	init( RESOURCE_CONSTRAINED_MAX_BACKOFF,       30.0 );
+	init( PROXY_COMMIT_OVERHEAD_BYTES,              23 ); //The size of serializing 7 tags (3 primary, 3 remote, 1 log router) + 2 for the tag length
```

Comment:
My thought was that we may want to make it zero again in case there is a problem with the change. In retrospect, it is hard to imagine a problem caused by the batching could which would require changing it.